app/models/comment.rb

```diff
@@ -0,0 +1,22 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: :comment_counter
```

Comment:
- [ Required ] I think it would be better if you follow the project requirements when updating the `comment_counter` field I know that you can update the `comment_counter` field using `counter_cache` but according to the project requirements, this field should be updated using the `update_comments_counter` method so let's follow the project guidelines. Please see the comment I've left on the `update_comment_counter` method.

## Project requirements
![Screenshot from 2023-07-20 00-41-40](https://github.com/Elhambasir/blog-app/assets/16392046/81a9e4ad-3e3b-42ad-a3e2-5a39f7020bb8)